nlmod/gwf/recharge.py

```diff
@@ -154,15 +200,318 @@ def ds_to_evt(gwf, ds, pname="evt", nseg=1, surface=None, depth=None, **kwargs):
+        groundwater flow model.
+    ds : xr.DataSet
+        dataset containing relevant model grid information
+    mask : xr.DataArray
+        data array containing mask, uzf is only added where mask is True
+    pname : str, optional
+        package name. The default is 'uzf'.
+    surfdep : float, str or array-like
+        surfdep is the surface depression depth of the UZF cell. When passed as string,
+        the array is obtained from ds. The default is 0.05 m.
+    vks : float, str or array-like
+        vks is the saturated vertical hydraulic conductivity of the UZF cell. This value
+        is used with the Brooks-Corey function and the simulated water content to
+        calculate the partially saturated hydraulic conductivity. When passed as string,
+        the array is obtained from ds. The default is 'kv'.
+    thtr : float, str or array-like
+        thtr is the residual (irreducible) water content of the UZF cell. This residual
+        water is not available to plants and will not drain into underlying aquifer
+        cells. When passed as string, the array is obtained from ds. The default is
+        0.1.
+    thts : float, str or array-like
+        thts is the saturated water content of the UZF cell. The values for saturated
+        and residual water content should be set in a manner that is consistent with the
+        specific yield value specified in the Storage Package. The saturated water
+        content must be greater than the residual content. When passed as string, the
+        array is obtained from ds. The default is 0.3.
+    thti : float, str or array-like
+        thti is the initial water content of the UZF cell. The value must be greater
+        than or equal to the residual water content and less than or equal to the
+        saturated water content. When passed as string, the array is obtained from ds.
+        The default is 0.1.
+    eps : float, str or array-like
+        eps is the exponent used in the Brooks-Corey function. The Brooks-Corey function
+        is used by UZF to calculated hydraulic conductivity under partially saturated
+        conditions as a function of water content and the user-specified saturated
+        hydraulic conductivity. Values must be between 3.5 and 14.0. When passed as
+        string, the array is obtained from ds. The default is 3.5.
+    landflag : xr.DataArray, optional
+        A DataArray with integer values, set to one for land surface cells indicating
+        that boundary conditions can be applied and data can be specified in the PERIOD
+        block. A value of 0 specifies a non-land surface cell. Landflag is set to one
+        for the most upper active layer in each vertical column (determined form ds)
+        when it is None. The default is None.
+    finf :float, str or array-like
+        The applied infiltration rate of the UZF cell (:math:`LT^{-1}`). When passed as
+        string, the array is obtained from ds. The default is "recharge".
```

Comment:
Do we want to name this "recharge"  by default or should it be precipitation? Recharge implies we already include evaporation somehow?

Reply:
We can also set the default to None so the user has to decide what data should be used?